### PR TITLE
Updated artifact filter test to support Python 3.3+

### PR DESCRIPTION
**Pull request by Onager**

### CODE REVIEW

File: tests/engine/artifact_filters.py [hunk 1]
```diff
@@ -186,7 +187,13 @@ def testBuildFindSpecsFromFileArtifact(self):
     self.assertEqual(len(find_specs), 1)
 
     # Location segments should be equivalent to \Windows\test_data\*.evtx.
-    path_segments = ['Windows', 'test\\_data', '.*\\.evtx']
+    if py2to3.PY_3:
```
Comment (joachimmetz) [inline tests/engine/artifact_filters.py]: Shouldn't the check here be `Python version >= 3.3` ?

Comment (Onager) [reply tests/engine/artifact_filters.py]: We don't support any python version below 3.4, so that's not necessary.

Comment (joachimmetz) [reply tests/engine/artifact_filters.py]: I understand but if change the check we technically can remove the comment.

Comment (Onager) [reply tests/engine/artifact_filters.py]: I don't think it's worth the extra complexity. Changed the docstring a bit.

Comment (joachimmetz) [reply tests/engine/artifact_filters.py]: ack.

**Pull request merged**